wdl-ast/src/v1/expr.rs

```diff
@@ -1983,14 +2245,14 @@ impl Placeholder {
             .expect("should have a placeholder open token")
     }
 
-    /// Gets the option for the placeholder.
-    pub fn option(&self) -> Option<PlaceholderOption> {
-        child(&self.0)
+    /// Gets the options for the placeholder.
+    pub fn options(&self) -> impl Iterator<Item = PlaceholderOption> {
+        PlaceholderOption::children(&self.0)
     }
```

Comment:
This was previously intentionally finding the first placeholder child as only one placeholder option is legal; the validation explicitly uses `children` to validate that only one exists.

Is there a reason for this change?

Reply:
Have to defer to @claymcleod on this one

Reply:
Oh! Okay I was not aware of that. Great, we can revert this (I'll do that in a moment).